dbt/include/mysql5/macros/materializations/snapshot/snapshot.sql

```diff
@@ -59,7 +70,7 @@
                                    | rejectattr('name', 'equalto', 'DBT_UNIQUE_KEY')
                                    | list %}
 
-      {% do create_columns(target_relation, missing_columns) %}
+      {% do mysql5__create_columns(target_relation, missing_columns) %}
```

Comment:
This change (and the others like it) is not necessary. The `create_columns()` macro will automatically call the adapter's specific macro if one exists (which it now does).

Reply:
I've made this change myself so that we can get this merged.